include/net/fota_download.rst

```diff
@@ -3,13 +3,15 @@
 FOTA download
 #############
 
-The firmware over-the-air (FOTA) download library provides functions for downloading a firmware file as an upgrade candidate to what is enabled in the :ref:`lib_dfu_target` library.
+The firmware over-the-air (FOTA) download library provides functions for downloading a firmware file as an upgrade candidate to the DFU target that is used in the :ref:`lib_dfu_target` library.
 
 The library uses the :ref:`lib_download_client` to download the image.
-Once the download has been started, all received data fragments are passed to the :ref:`lib_dfu_target`. The :ref:`lib_dfu_target` takes care of where the upgrade candidate is stored depending on the image type being downloaded.
+Once the download has been started, all received data fragments are passed to the :ref:`lib_dfu_target` library.
+The :ref:`lib_dfu_target` library takes care of where the upgrade candidate is stored, depending on the image type that is being downloaded.
 
-When the download client sends the event indicating that the download has completed. The received firmware is tagged as an upgrade candidate, and the download client is instructed to disconnect from the server.
-The library then emits a `FOTA_EVT_DONE` callback event. When this is received the consumer of the library should issue a reboot to apply the upgrade.
+When the download client sends the event indicating that the download has completed, the received firmware is tagged as an upgrade candidate, and the download client is instructed to disconnect from the server.
+The library then sends an :cpp:enumerator:`AWS_FOTA_EVT_DONE <aws_fota::AWS_FOTA_EVT_DONE>` callback event.
```

Comment:
Is this the correct event?

Reply:
no the ` FOTA_EVT_DONE` is emitted from the `fota_download` library It does not relate to AWS

Reply:
Do you mean FOTA_DOWNLOAD_EVT_FINISHED then? I can't find any event called FOTA_EVT_DONE. ;)

Reply:
Ah yes `FOTA_DOWNLOAD_EVT_FINISHED` I'll change it.